src/pkynetics/__init__.py

```
@@ -6,15 +6,16 @@
-from . import synthetic_data
-from . import technique_analysis
+from . import (
+    data_import,
+    data_preprocessing,
+    model_fitting_methods,
+    model_free_methods,
+    result_visualization,
+    synthetic_data,
+    technique_analysis,
+)
+from .__about__ import __version__
```

Ruff (0.8.0)

`.__about__.__version__` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)